banks.go

```diff
@@ -2,8 +2,8 @@ package ghipu
 
 import "net/http"
 
-// Bank represents the schema of bank defined by khipu
-type Bank struct {
+// BankItem represents the schema of bank defined by khipu
+type BankItem struct {
```

Comment:
In the first version of Ghipu had declared the structure with this name, however, sebastian recommended me to change it